docs/manuals/source/Appendix/Howtos.rst

```diff
@@ -945,6 +945,144 @@ A very simple corresponding shell script (:command:`bpipe-restore.sh`) to the me
    cat - > /tmp/dump.sql
    exit 0
 
+
+.. _section-MigrationMysqlToPostgresql:
+
+Migrate |mysql| to |postgresql|
+---------------------------
+
+:index:`\ <single: Migration; MySQL; postgreSQL; bareos-dbcopy; dbcopy>`
+
+Since Bareos :sinceVersion:`19.0.0:` the use of |mysql| databases with
+Bareos is deprecated. Therefore Bareos provides a tool to conveniently copy the
+whole contents to a new |postgresql| database: :ref:`program-bareos-dbcopy`. This
+chapter describes how to do a migration using bareos-dbcopy.
+
+Make a backup of your old database
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. warning::
+
+   Make a backup of your old database before you start the migration process!
```

Comment:
A subsection with only an admonition is weird.

Reply:
The docs were probably not ready, yet.

Reply:
Ha, no. This was intended, because I wanted to point out to make a backup! This is only a subchapter and therefor it has only this headline and the warning.